cli/commands.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"github.com/knoebber/dotfile/file"
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"fmt"
+	"os"
+)
+
+const (
+	defaultDataDir  string = ".dotfile/"
+	defaultDataName string = "files.json"
+)
+
+// Dotfile depends on the system having the concept of a home directory.
+func getHome() string {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
```

Comment:
I'd rather not panic in any case. Could we bubble up the `err` using `errors.Wrap()` and returning a `(string, error)` instead?